resources/indexer_make_dag.py

```diff
@@ -6,14 +6,22 @@
 import re
 import subprocess
 
+LEVELS = {
+    "L2": "filtered/level2",
+    "L2P2": "filtered/level2pass2",
+    "PFFilt": "filtered/PFFilt",
+    "PFDST": "unbiased/PFDST",
+    "PFRaw": "unbiased/PFRaw",
+}
 
-def _get_dirpaths():
-    years = [str(y) for y in range(2007, 2021)]
+
+def _get_dirpaths(begin_year, end_year, level):
+    years = [str(y) for y in range(begin_year, end_year)]
     dirs = [d for d in os.scandir(os.path.abspath('/mnt/lfs6/exp/IceCube')) if d.name in years]
```

Comment:
Do you want to make this a configurable? Or leave it with the hard-coded path: `/mnt/lfs6/exp/IceCube`?

Reply:
@ric-evans Did you want to make this root directory something you can specify from the command-line?

Reply:
Sure, I'll ad that as an option.